ultravox/model/ultravox_processing.py

```diff
@@ -175,3 +200,6 @@ def model_input_names(self):
         tokenizer_input_names = self.tokenizer.model_input_names
         audio_processor_input_names = self.audio_processor.model_input_names
         return list(set(tokenizer_input_names + audio_processor_input_names))
+
+
+transformers.AutoProcessor.register(UltravoxConfig, UltravoxProcessor)
```

Comment:
Nice! We should presumably remove the commented out bit in the other file too :)